app/code/community/Riskified/Full/Model/Observer/Order/Creditmemo/Save.php

```diff
@@ -57,15 +57,8 @@ public function handleCreditmemoSave(
             $reason
         );
 
-        $request = Mage::getModel('full/request_order_creditmemo_save');
-
-        try {
-            $response = $request->sendRequest($payload);
-        } catch (\Exception $e) {
-            Mage::helper('full/log')->log(
-                'handleCreditmemoSave(): ' . PHP_EOL . $e->getMessage()
-            );
-        }
+        $helper = Mage::helper('full/order');
+        $helper->postOrder($payload, 'refund');
```

Comment:
Action codes has been replaced with constants.

I can not use exactly the Model\Order because the objects do not allow to set fields which I need to pass. If you mean to use the models from sdk Model namespace, so I am already using them in the helper.